pkgs/development/tools/misc/texinfo/default.nix

```diff
@@ -0,0 +1,110 @@
+{
+  lib,
+  stdenv,
+  buildPackages,
+  updateAutotoolsGnuConfigScriptsHook,
+  fetchurl,
+  perl,
+  xz,
+  libintl,
+  libiconv,
+  bash,
+  gnulib,
+  gawk,
+  freebsd,
+  ncurses,
+  procps,
+  # we are a dependency of gcc, this simplifies bootstraping
+  interactive ? false,
+}:
+
+let
+  meta = {
+    description = "The GNU documentation system";
+    homepage = "https://www.gnu.org/software/texinfo/";
+    changelog = "https://git.savannah.gnu.org/cgit/texinfo.git/plain/NEWS";
+    license = lib.licenses.gpl3Plus;
+    platforms = lib.platforms.all;
+    maintainers = with lib.maintainers; [ oxij ];
+
+    longDescription = ''
+      Texinfo is the official documentation format of the GNU project.
+      It was invented by Richard Stallman and Bob Chassell many years
+      ago, loosely based on Brian Reid's Scribe and other formatting
+      languages of the time.  It is used by many non-GNU projects as
+      well.
+
+      Texinfo uses a single source file to produce output in a number
+      of formats, both online and printed (dvi, html, info, pdf, xml,
+      etc.).  This means that instead of writing different documents
+      for online information and another for a printed manual, you
+      need write only one document.  And when the work is revised, you
+      need revise only that one document.  The Texinfo system is
+      well-integrated with GNU Emacs.
+    '';
+    mainProgram = "texi2any";
+  };
+  make-texinfo = (import ./make-texinfo.nix) {
+    inherit
+      lib
+      stdenv
+      buildPackages
+      updateAutotoolsGnuConfigScriptsHook
+      fetchurl
+      perl
+      xz
+      libintl
+      libiconv
+      bash
+      gnulib
+      gawk
+      freebsd
+      ncurses
+      procps
+      meta
+      interactive
+      ;
+  };
+in
+{
+  texinfo_4_13 = stdenv.mkDerivation (finalAttrs: {
```

Comment:
Another `texinfo-4.13`? What about the file above?

Reply:
I think the `texinfo-4.13` files was re-created when I rebased this PRs branch onto staging again… I'll have another go at this PR and will delete the file from git in the process